zipkin-server/src/main/resources/zipkin-server.yml

```diff
@@ -58,6 +58,8 @@ zipkin:
       index-cache-ttl: ${CASSANDRA_INDEX_CACHE_TTL:60}
       # how many more index rows to fetch than the user-supplied query limit
       index-fetch-multiplier: ${CASSANDRA_INDEX_FETCH_MULTIPLIER:3}
+      # Using ssl for connection, rely on Keystore
+      use-ssl: ${CASSANDRA_USE_SSL:false}
```

Comment:
Did it in https://github.com/openzipkin/zipkin/pull/1401/commits/75d19a6b95cae017207d8a767418e78bef1c34c7